[2025-06-22T03:21:12.360+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-22T03:21:12.401+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment manual__2025-06-22T03:21:09.754151+00:00 [queued]>
[2025-06-22T03:21:12.425+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment manual__2025-06-22T03:21:09.754151+00:00 [queued]>
[2025-06-22T03:21:12.426+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-22T03:21:12.446+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): validate_environment> on 2025-06-22 03:21:09.754151+00:00
[2025-06-22T03:21:12.455+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4503) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-22T03:21:12.458+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'validate_environment', 'manual__2025-06-22T03:21:09.754151+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpr2qescps']
[2025-06-22T03:21:12.460+0000] {standard_task_runner.py:72} INFO - Started process 4504 to run task
[2025-06-22T03:21:12.460+0000] {standard_task_runner.py:105} INFO - Job 142: Subtask validate_environment
[2025-06-22T03:21:12.530+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.validate_environment manual__2025-06-22T03:21:09.754151+00:00 [running]> on host 246f02c9dee3
[2025-06-22T03:21:12.646+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Jasjot Parmar' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-06-22T03:21:09.754151+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-22T03:21:09.754151+00:00'
[2025-06-22T03:21:12.648+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-22T03:21:12.684+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-06-22T03:21:12.687+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', ' \n        echo "Validating environment..."\n        test -f /app/config.yaml &&\n        test -f /app/.env &&\n        echo "Environment is valid!"\n        ']
[2025-06-22T03:21:12.703+0000] {subprocess.py:99} INFO - Output:
[2025-06-22T03:21:12.706+0000] {subprocess.py:106} INFO - Validating environment...
[2025-06-22T03:21:12.706+0000] {subprocess.py:106} INFO - Environment is valid!
[2025-06-22T03:21:12.707+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-06-22T03:21:12.743+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-22T03:21:12.744+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=validate_environment, run_id=manual__2025-06-22T03:21:09.754151+00:00, execution_date=20250622T032109, start_date=20250622T032112, end_date=20250622T032112
[2025-06-22T03:21:12.805+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-22T03:21:12.847+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-22T03:21:12.849+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
